cmd/helm/status.go

```diff
@@ -80,7 +80,7 @@ func newStatusCmd(cfg *action.Configuration, out io.Writer) *cobra.Command {
 			// strip chart metadata from the output
 			rel.Chart = nil
 
-			return outfmt.Write(out, &statusPrinter{rel, false, client.ShowDescription, client.ShowResources})
+			return outfmt.Write(out, &statusPrinter{rel, false, client.ShowDescription, false, false})
```

Comment:
should this be

```suggestion
			return outfmt.Write(out, &statusPrinter{rel, false, client.ShowDescription, client.ShowResources, false})
```
or maybe even add HideNotes to the status command so you can use client.HideNotes in there.